### ISSUE TITLE
Border around the tabs section

### ISSUE THREAD

**Issue by jcesguerram** [xhuajin/obsidian-tabs#59]
This is what the new Tabs Panel plugin looks like:

![image](https://github.com/user-attachments/assets/b1a22173-59fa-446b-b0fc-d12ca9977353)

As you can see, I was trying to change the size of the text in the first tab, but it's not possible. This is something in which your plugin is better (something important!)
I was wondering though if you could add a border to the tabs pane to your plugin. I think this is a nice design touch from Tabs Panel

**Comment by xhuajin** [comment]
I tested the modified text font size and it work. Can you show me the grammar you use?
![image](https://github.com/user-attachments/assets/1a494076-945b-4b16-8e96-da769dc8d4ab)

I don't quite understand "add a border to the tabs pane". Can you give me a further explanation?

**Comment by jcesguerram** [comment]
No, that is from a different plugin. What I'm saying is is that in your plugin the modified font size works!
What I'm showing you is that this other plugin has a defined border and this grey area inside, and iI think it looks really good. Maybe you can consider that for your plugin
